pkg/scaffold/resource/resource.go

```diff
@@ -16,144 +16,42 @@ limitations under the License.
 
 package resource
 
-import (
-	"fmt"
-	"regexp"
-	"strings"
-
-	"github.com/gobuffalo/flect"
-)
-
 // Resource contains the information required to scaffold files for a resource.
 type Resource struct {
-	// Namespaced is true if the resource is namespaced
-	Namespaced bool
+	// Namespaced is true if the resource is namespaced.
+	Namespaced bool `json:"namespaces,omitempty"`
```

Comment:
Could it be empty? If not, then : 

```suggestion
	Namespaced bool `json:"namespaces"`
```

Reply:
False values will be considered empty and thus not passed to the JSON or YAML files, which makes if a little bit more memory efficient. If default values are true this doesn't work